query-results/template.yaml

```diff
@@ -25,6 +25,7 @@ Conditions:
       !Equals [!Ref Environment, integration]
     ]
   IsProduction: !Equals [!Ref Environment, production]
+  TestEnvironment: !Not [Condition: NotTestEnvironment]
```

Comment:
Should this just be TestEnvironment?

Reply:
The condition we're creating here is just TestEnvironment.  And we're setting that to the opposite of the existing NotTestEnvironment condition.